frontend/packages/dev-console/src/components/topology/topology-types.ts

```diff
@@ -100,6 +100,7 @@ export interface ConnectedWorkloadPipeline {
 export interface WorkloadData {
   url?: string;
   editUrl?: string;
+  cheEnabled?: boolean;
```

Comment:
I wonder if we're better off having a `cheUrl` here and in the react component check if `cheUrl` is present then use it and the che icon, else do the status quo.

Reply:
editUrl is already determining the correct url to pick, this cheEnabled variable is used only to pick the right icons.